xbmc/addons/kodi-addon-dev-kit/include/kodi/addon-instance/Inputstream.h

```diff
@@ -258,7 +270,7 @@ namespace addon
     * @param enable true for enable, false for disable
     * @remarks
     */
-    virtual void EnableStream(int streamid, bool enable) { }
+    virtual void EnableStream(int streamid, bool enable) = 0;
```

Comment:
Maybe I change them above, normally I not like to use a whole strurcture as return value, want it more to have as `void GetStreamIds(INPUTSTREAM_IDS& ids);` where memset become done from Kodi.
Was taken like them from the old interface.